js/common/util_test.go

```diff
@@ -11,6 +11,7 @@ import (
 
 func TestThrow(t *testing.T) {
 	t.Parallel()
+	t.Skip("Until we remvoe GoError")
```

Comment:
```suggestion
	t.Skip("Until we remove GoError")
```

Should we link https://github.com/grafana/k6/issues/2049?

Reply:
I don't understand why the test is skipped, but to be honest, I am not sure precisely what it's testing :sweat_smile: Especially the second half, which has a minor bug - `fn1 is invalid` should probably be `fn2 is invalid`

Reply:
It is testing that the reported error will *not* have `GoError: ` in front

Reply:
> Should we link https://github.com/grafana/k6/issues/2049?

I am not personally all that big of a fan of back referencing, but if you want to :shrug:

Reply:
TBH, now @na-- mentioned it, I think would be better to just refactor and re-enable this test, in the end, it is also testing that the `Throw` function is working as expected and we still have it.

I think we could have two cases:
- one in case of `*goja.Exception`
- and one in the else path

Reply:
[0503fcb](https://github.com/grafana/k6/pull/2739/commits/0503fcb06d2d80efed3f03bb073fcb4f323ae8f9)